debug.js

```diff
@@ -141,7 +141,7 @@ function enable(namespaces) {
 
   for (var i = 0; i < len; i++) {
     if (!split[i]) continue; // ignore empty strings
-    namespaces = split[i].replace(/\*/g, '.*?');
+    namespaces = split[i].replace(/[\\^$+?.()|[\]{}]/g, '\\$&').replace(/\*/g, '.*?');
```

Comment:
I looked at that, but `*` would be escaped by that (or any such) library -- you'll note I'm not escaping it here so the existing wildcard replace will work as-is.  It's possible to work around this issue, but for a 1-liner it seemed like overkill.